index.bs

```diff
@@ -85,8 +88,8 @@ The DeviceOrientationSensor Interface {#deviceorientationsensor-interface}
 <pre class="idl">
   [Constructor(optional SensorOptions sensorOptions)]
   interface DeviceOrientationSensor : Sensor {
-    void populateQuaternion(Float32Array array);
-    void populateMatrix(Float32Array array);
+    void populateQuaternion(Float32Array buffer);
```

Comment:
Yes. But this is also something we want to do, no? So we need a consistent approach here, else we'll end up with different data representations depending on how the data is collected.